tools/RadioGUI/gui.log

```diff
@@ -0,0 +1,1887 @@
+2022-04-15 20:12:56,856 | root | INFO | GUI started, opening serial port COM5
```

Comment:
don't commit the log file, in fact the .gitignore should be set up to prevent this from showing up.